mon-pix/app/styles/pages/_update-expired-password-form.scss

```diff
@@ -1,7 +1,7 @@
 .update-expired-password-form-page {
   display: flex;
   justify-content: center;
-  background: $pix-blue-gradient;
+  background:  linear-gradient(135deg, #388AFF 0%, #985FFF 100%); /* TODO: $pix-purple-gradient; doesn't work here, find why */
```

Comment:
On a le droit de merger des `TODO` 😨 ?

Reply:
Le but n'est pas que ça reste, c'est un AFH explicite !

Reply:
AFH ? 🤔

Reply:
Résolu, les gradients avaint été renommés dans la PR #1244

Reply:
Ça me dit pas ce qu'est un AFH 😢

Reply:
Effectivement, AFH n'est pas clair. C'est une demande d'aide : Ask For Help.

Et ça peut être plus qu'une simple demande d'aide car ça fait partie des Core Protocols (je ne sais pas si c'était ton intention Pierre).

J'aime bien ce blog sur le sujet :
- https://bloculus.com/core-protocols-7-ask-for-help/

Reply:
J'aime les endroits où des core protocols se balladent dans les PR :smile: 
Désolé Yvonnick pour le jargon